src/amo/api/index.js

```diff
@@ -1,5 +1,5 @@
 /* @flow */
-/* global fetch, window */
```

Comment:
This is needed because of https://github.com/sindresorhus/globals/commit/508d431688787f45f9a12225d9c3410cd0a12a1c - `globals` is part of the packages updated in this PR (from `13.11.0` to `13.17.0` - that commit has been present since `13.14.0`)